UefiPayloadPkg/Library/PayloadEntryHobLib/HobLib.inf

```diff
@@ -36,4 +36,5 @@
 [Guids]
   gEfiHobMemoryAllocModuleGuid
   gEfiHobMemoryAllocStackGuid
+  gUniversalPayloadDeviceTreeGuid

```

Comment:
(I expect this to be used at some point, but I can't see it in this patch yet?.)

Reply:
fyi Benjamin means that this might be used in the later commit so it might not belong to this commit ;)

Reply:
yes, we expect some platform payload driver might want to directly consume FDT table so here is providing the data pointer for such usage model.

Reply:
Agree we can remove gUniversalPayloadDeviceTreeGuid from HobLib.inf as it was not  consumed by this libray. The expecation is some of the payload drivers might consume this in the future.

Reply:
Todo.

Reply:
resolved.